massa-node/base_config/config.toml

```diff
@@ -307,6 +307,8 @@
     max_in_connections = 250
     # Cooldown before testing again old peer
     test_oldest_peer_cooldown = 720000
+    # Rate limitation on the data streams (per second)
+    rate_limit = 2097152    # 2 Mo / secs
```

Comment:
Mo is in French (o = octet), use byte "B" (1 byte = 8 bits).

Also, those are not megabytes (1 MB = 1000000 B) but mebibytes (1 MiB = 1048576 B).